Wikipedia/Code/FakeProgressLoading.swift

```diff
@@ -0,0 +1,14 @@
+
+import Foundation
+
+protocol FakeProgressLoading {
+    var fakeProgressController: FakeProgressController { get }
+}
+
+extension TalkPageContainerViewController: FakeProgressLoading {
+    
+}
+
+extension TalkPageReplyListViewController: FakeProgressLoading {
```

Comment:
Is there a reason the empty conformances are declared here and not in the files?

Reply:
@nambatee nope no reason, I have moved these to the files.